CHANGELOG.md

```diff
@@ -2,15 +2,17 @@
 
 ## Unreleased
 
+### Fixes
+
+- Android: Fix for SIGSEGV reported by Sentry when a NullReferenceException is thrown in Release mode ([#3694](https://github.com/getsentry/sentry-dotnet/pull/3694))
```

Comment:
Is that this PR or another PR? The title talks about attachments but the changelog talks about fixing a `NullReferenceException`...

Reply:
it's not from here, I have no idea how it ended up here

Reply:
similarly it's coming from me

either way it's fixed now